apps/src/netsim/NetSimRemoteNodeSelectionPanel.html.ejs

```diff
@@ -0,0 +1,176 @@
+<%
+var utils = require('../utils');
+var _ = utils.getLodash();
+var i18n = require('../../locale/current/netsim');
+var NodeType = require('./netsimConstants').NodeType;
+
+
+/**
+ * @typedef {Object} rowMetadata
+ * @property {number} nodeID
+ * @property {string} classAttr
+ * @property {string} displayName
+ * @property {string} status
+ */
+
+/**
+ * @param {NetSimNode} node
+ * @returns {boolean}
+ * @private
+ */
+var isSelectedNode = function (node) {
```

Comment:
Sure; putting this together, I was just tempted to pass the controller instance into the ejs template so that it could call utility methods like this.  Is that considered bad form?